package.json

```diff
@@ -15,8 +15,8 @@
   "author": "Gal Schlezinger <gal@spitfire.co.il>",
   "license": "MIT",
   "devDependencies": {
-    "@glennsl/bs-jest": "^0.4.2",
-    "bs-platform": "^3.1.5"
+    "@glennsl/bs-jest": "^0.4.9",
+    "bs-platform": "^7.0.1"
```

Comment:
Do you think we should add it as a peer dependency too so it'll warn if you use with an incompatible `bs-platform`?

Reply:
Yes. I do think that but I don't know why it matters exactly so I figured I would let you mention it. I saw @fhammerschmidt mention on in this [article on dev.to](https://dev.to/fhammerschmidt/what-s-new-in-bucklescript-7-1bf4) that he had upgraded some code with 7 but released as 5. So maybe we do something like that?

Reply:
I did try using `Js.Re.test_` and then it kept asking for other changes which I didn't have the bandwidth to keep chasing down since testing is not yet a thing I have deep dived.